client/js/screens/Home/Home.js

```diff
@@ -53,16 +58,46 @@ class Home extends Component {
     this.setState({showPicker: false});
   };
 
+  onScroll = e => {
+    const offset = e.nativeEvent.contentOffset;
+
+    if (offset.y > 88 && !this.state.isAnimated) {
+      this.setState({isAnimated: true});
+      this.animateHeader();
+    } else if (offset.y <= 88 && this.state.isAnimated) {
+      this.setState({
+        isAnimated: false,
+      });
+
+      this.animateHeader();
+    }
+  };
+
+  animateHeader = () => {
+    Animated.timing(this.state.headerAnimation, {
+      duration: 500,
+      toValue: this.state.isAnimated ? {x: 0, y: -88} : {x: 1, y: 0},
+    }).start();
+  };
```

Comment:
what is 88? Please don't put magic numbers in your code. Use %s if possible to create variabels/consts.

Reply:
It's the height of the header navigation and I replaced it with the variable~